tmk_core/protocol.mk

```diff
@@ -6,6 +6,13 @@ SRC +=	\
 
 SHARED_EP_ENABLE = no
 MOUSE_SHARED_EP ?= yes
+
+ifeq ($(strip $(GLOBE_KEY_ENABLE)), yes)
+    OPT_DEFS += -DGLOBE_KEY_ENABLE
+    # The globe key requires SHARED_EP
+    SHARED_EP_ENABLE = yes
+endif
```

Comment:
On closer inspection this doesn't really do anything since the keycode is implemented as just another Consumer usage under `EXTRAKEY_ENABLE`, which already sets `SHARED_EP_ENABLE = yes`.

Did you mean to set `KEYBOARD_SHARED_EP` instead? I can imagine the report descriptor containing the Globe key might be required to be on the same interface as the keyboard. I haven't played with this stuff in a while but that could get the F-keys working. Otherwise, this file doesn't need to be touched at all.

Reply:
Oh, you are right, my bad! Indeed, I mean to set KEYBOARD_SHARED_EP instead.
Let me correct that

Reply:
TBH I'm unsure if we truly need a `GLOBE_KEY_ENABLE` anyway as all it's doing is hiding what's actually going on - I think I would prefer to just call out `KEYBOARD_SHARED_EP` directly in the documentation, especially if the Globe key doesn't work at all without being on the same interface as the keyboard report.

Reply:
I agree this flag is pretty trivial, but I thought it would be more self-explanatory for users who only create layouts or even keyboard builders than the more obscure `KEYBOARD_ENABLE_EP`.

Reply:
The issue I see is that `GLOBE_KEY_ENABLE = yes` without `EXTRAKEY_ENABLE = yes` doesn't actually enable the Globe key by itself, but it would still combine the keyboard report descriptor into the shared endpoint. So the functionality here can potentially be non-obvious, and I'd rather be more explicit about it.

Reply:
I see. Well, I don't have a strong opinion here. I'm fine editing the documentation to make it more obvious for users. I would only need you to guide me where you'd like that done, and I can work on it tomorrow.